src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -87,6 +87,23 @@ PARAM_DEFINE_FLOAT(FW_L1_DAMPING, 0.75f);
  */
 PARAM_DEFINE_FLOAT(FW_THR_CRUISE, 0.6f);
 
+/**
+ * Scale throttle by pressure
+ *
+ * This parameter enables throttle scaling based on barometric pressure
+ * When flying without airspeed sensor this will help to keep a constant performance over large altitude ranges.
+ *
+ * A value of 0 will disable scaling.
+ * A value of 1 will add 1% throttle for every 1% of pressure descrease relative to standard pressure (1013.25)
```

Comment:
Param description out of date.

Reply:
Whats out of date about it? i just rewrote it